coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -2383,7 +2383,8 @@ def _parse_positional_args(context, node) -> Tuple[Var]:
         # PyTorch can have empty tensor, which is then ignored
         # However, CoreML does not allow such empty tensor, so remove them now
         if np.any([is_tensor_empty(x) for x in xs]):
-            xs = [x for x in xs if not is_tensor_empty(x)]
+            filtered_xs = [x for x in xs if not is_tensor_empty(x)]
+            xs = filtered_xs if filtered_xs else [xs[0]]
```

Comment:
Great PR ...
nit: I think it will be better to do `xs = filtered_xs if len(filtered_xs) == 0 else [xs[0]]` to make the code more clear

Reply:
I've updated the code to use len() in the condition, please have another look, thank you!